com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/dependency/CopybookDependencyService.java

```diff
@@ -24,11 +26,14 @@
  * management
  */
 public interface CopybookDependencyService {
+
+  void addCopybookInDepFile(RequiredCopybookEvent event, String requiredCopybookName);
+
+  boolean isFileInDidOpen(RequiredCopybookEvent event);
```

Comment:
Yeah we already agree that is better hide this implementation detail, the problem is that I need PowerMock to do that, and I opened an issue to track the activity to make it private again!

Reply:
For the naming I could change it no problem but is just check that the enum type for the document is DID_OPEN, I don't see any relation with update the dependency. Furthermore we can in the future use this method for checks not dependent on the dependency file update. Let me know if I miss something from the picture

Reply:
There is a little difference between the naming by method logic and by its purpose. The first one describes how the method works, and the second one describes the useful outcome of the method. Here I don't really care that processing type was used to invoke this event, the only important thing is if I need to apply some changes according to that.

Reply:
Yes Andrei but the purpose of this method is only to apply a check, and when it will be private (when Powermock will be finally part of our project) it will became private, with nothing made available for the client ;)